engine/frontend/src/component/log/useWindowResize.js

```diff
@@ -0,0 +1,18 @@
+import { useState, useLayoutEffect } from "react";
```

Comment:
@Peeeekay this is almost identical to this file:
https://github.com/kurtosis-tech/kurtosis/blob/main/engine/frontend/src/utils/windowheight.js
Can we refactor it to become one file